### PR TITLE
Docs: Missing docs added

### PR DESCRIPTION
**Pull request by ShilpaVernekar**
### Describe your changes:
Re-adding the missing docs and images
1. Conversation Threads
2. Tasks

Docs that were lost in the last PR have been re-added

v1.4
/connectors/ingestion/workflows/metadata/incremental-extraction
/connectors/ingestion/workflows/metadata/incremental-extraction/bigquery
/connectors/ingestion/workflows/metadata/incremental-extraction/redshift
/connectors/ingestion/workflows/metadata/incremental-extraction/snowflake

v1.1, v1.2, v1.3, v1.4
/connectors/ingestion/workflows/metadata/filter-patterns
/connectors/ingestion/workflows/metadata/filter-patterns/database

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-docs/images/v1.3/how-to-guides/collaboration/task.png
Comment (aniketkatkar97) [inline openmetadata-docs/images/v1.3/how-to-guides/collaboration/task.png:1]: Can we please have all the images in .webp format instead of .png?

Comment (ShilpaVernekar) [reply openmetadata-docs/images/v1.3/how-to-guides/collaboration/task.png:1]: Would it be okay if I replace these images later?

**Comment by ShilpaVernekar** [comment]
Hi @aniketkatkar97 - Please check. Replaced images

File: openmetadata-docs/images/v1.1/how-to-guides/collaboration/task.png
Comment (aniketkatkar97) [inline openmetadata-docs/images/v1.1/how-to-guides/collaboration/task.png:1]: It looks like we missed removing this image. It is the same as task.webp right?

File: openmetadata-docs/images/v1.2/how-to-guides/collaboration/task.png
Comment (aniketkatkar97) [inline openmetadata-docs/images/v1.2/how-to-guides/collaboration/task.png:1]: Same here

File: openmetadata-docs/images/v1.3/how-to-guides/collaboration/task.png
Comment (aniketkatkar97) [inline openmetadata-docs/images/v1.3/how-to-guides/collaboration/task.png:1]: same here

File: openmetadata-docs/images/v1.4/how-to-guides/collaboration/task.png
Comment (aniketkatkar97) [inline openmetadata-docs/images/v1.4/how-to-guides/collaboration/task.png:1]: same here

**Review by aniketkatkar97 — COMMENTED** [review]
@ShilpaVernekar left few comments

**Comment by ShilpaVernekar** [comment]
@aniketkatkar97 - Done. removed the task.png files

**Review by aniketkatkar97 — APPROVED** [review]

**Pull request merged**